xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminTranslations.xml

```diff
@@ -78,7 +78,7 @@ XWiki.XWikiPreferences_showattachments.hint=Display the attachments section in t
 XWiki.XWikiPreferences_showhistory.hint=Display the history section in the page layout.
 XWiki.XWikiPreferences_showinformation.hint=Display the information section in the page layout.
 XWiki.XWikiPreferences_title.hint=Define the page title format as seen by browsers.
-XWiki.XWikiPreferences_meta.hint=Define the meta data added to each page, specifying page description, author, keywords etc.
+XWiki.XWikiPreferences_meta.hint=Define the meta data added to each page, specifying page description, author, keywords etc. It is advised to use #template('meta.vm') as part of the content to benefit easily from future improvements of the default meta definition.
```

Comment:
I'm wondering if even the first part should be rewritten to indicate that the default implementation is located in `meta.vm`, and then explain how to add something before/after the standard content.

Reply:
I updated the hint with `Specifies the metadata that is included in the HTML header of each page, including the page description, author, and keywords. The default meta.vm template is used when the meta field is blank. Use #template('meta.vm') as part of the field content to benefit from future enhancements to the default meta definition.`

Reply:
Sounds good.